proto/frequenz/api/reporting/v1/reporting.proto

```diff
@@ -182,6 +182,19 @@ message SimpleAggregatedMetricValue {
   frequenz.api.common.v1.metrics.SimpleMetricValue sample = 2;
 }
 
+// Filter for sources for metrics, specified for a single metric at a time.
+// Specifying a metric here will only stream data for that metric where data
+// can be found tagged with the given source names.
+// This is optional, and if omitted, the API will default to serving untagged metrics.
```

Comment:
This line somehow reads that "if omitted, the API will serve metrics that do not have any sources tagged to them".

Also, since this is a message definition, somehow stating that it can be optional does not fit here, since it depends on how this is being used.

Shall we just remove this line?

Reply:
Sounds good to me